### DISCUSSION TITLE
Reference data for AOD variables

### DISCUSSION THREAD

**Discussion by adagj** [NorESMhub/noresm3_dev_simulations#47] [Oslo-Aero]
Hi,
I'm trying to add aerosol diagnostics to ADF, but I need observations / reference data to make comparisons. Which datasets can I use?

No reference data found for variable `AODVIS`, global lat/lon mean plotting skipped.
No reference data found for variable `D550_BC`, global lat/lon mean plotting skipped.
No reference data found for variable `D550_DU`, global lat/lon mean plotting skipped.
No reference data found for variable `D550_POM`, global lat/lon mean plotting skipped.
No reference data found for variable `D550_SO4`, global lat/lon mean plotting skipped.
No reference data found for variable `D550_SS`, global lat/lon mean plotting skipped.

@oyvindseland @MichaelSchulzMETNO

**Comment by andrearosendahl** [comment]
I use AeronetSDAV3Lev2.daily observation when looking at D550_DU, but I first divide D550_DU/DAYFOC so it is comparable with the od550dust variable in the observation set.

**Reply by adagj** [reply]
Thanks! 
Do you have a path to those data?

**Reply by andrearosendahl** [reply]
I copied it over from lustre to the NS9560K project. so on nird: /projects/NS9560K/diagnostics/AeroVal/obs/AEROCOM_OBSDATA/Aeronet.SDA.V3L2.0.daily/renamed/

It has data from several stations, so hopefully that works when using ADF.
